### PR TITLE
Hang Yourself ♡

### PR DESCRIPTION
**Pull request by Litberries**
## About The Pull Request

adds noose

## Why You Should Do it

YOUR LIFE ZERVES ZERO PURPOSE, YOU SHOULD KILL YOURSELF **NOW!**

## Changelog
:cl:
add: Noose

/:cl:

### CODE REVIEW

File: code/game/objects/structures/noose.dm [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+	name = "noose"
+	desc = "Hang in there!"
+	icon_state = "noose"
+	icon = 'icons/obj/noose.dmi'
+	layer = FLY_LAYER
+	flags_1 = NODECONSTRUCT_1
+	var/mutable_appearance/overlay
+
+/obj/structure/chair/noose/attackby(obj/item/W, mob/user, params)
+	if(W.tool_behaviour == TOOL_WIRECUTTER)
```
Comment (Gboster-0) [inline code/game/objects/structures/noose.dm:18]: can save a layer of indentation here by reverting the check
```suggestion
	if(W.tool_behaviour != TOOL_WIRECUTTER)
		return ..()
	[the rest of code 1 layer thinner]
```

Comment (Litberries) [reply code/game/objects/structures/noose.dm]: done

File: code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -703,6 +703,13 @@
 				/obj/item/stack/rods = 1)
 	category = CAT_MISC
 
+/datum/crafting_recipe/noose
+	name = "Noose"
+	result = /obj/structure/chair/noose
+	reqs = list(/obj/item/stack/cable_coil = 20)
+	time = 20 //i mean cmon you gotta attach it to the damn ceiling (fuck you cox its REAAL)
```
Comment (Coxswain-Navigator) [inline code/datums/components/crafting/recipes.dm:710]: Nothing stops you from building this outside in the citymaps.

Comment (Litberries) [reply code/datums/components/crafting/recipes.dm:710]: whats the issue with that

File: code/datums/components/crafting/recipes.dm [hunk 3]
```diff
@@ -703,6 +703,13 @@
 				/obj/item/stack/rods = 1)
 	category = CAT_MISC
 
+/datum/crafting_recipe/noose
+	name = "Noose"
+	result = /obj/structure/chair/noose
+	reqs = list(/obj/item/stack/cable_coil = 20)
+	time = 20 //womp womp
```
Comment (Gboster-0) [inline code/datums/components/crafting/recipes.dm:710]: i just realized you did the forbidden trait rework thing, we have the SECONDS and MINUTES defines for numbers that are supposed to represent time
```suggestion
	time = 2 SECONDS //womp womp
```

File: code/game/objects/structures/noose.dm [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+			M.Knockdown(60)
+		unbuckle_all_mobs(force=1)
+		M.pixel_z = initial(M.pixel_z)
+		pixel_z = initial(pixel_z)
+		M.pixel_x = M.base_pixel_x
+		pixel_x = base_pixel_x
+		add_fingerprint(user)
+
+/obj/structure/chair/noose/user_buckle_mob(mob/living/carbon/human/M, mob/user, check_loc = TRUE)
+	//if(!in_range(user, src) || user.stat || user.restrained() || !iscarbon(M))
```
Comment (Gboster-0) [inline code/game/objects/structures/noose.dm:86]: why is this commented out?

Comment (Litberries) [reply code/game/objects/structures/noose.dm:86]: i forgot to delete it

**Comment by Litberries** [comment]
I'm just gonna make a new PR

**Pull request closed without merging**